test/swe2d/test_thacker.py

```diff
@@ -0,0 +1,60 @@
+# Test for wetting-drying scheme functionality, using Thacker test case
+# Test case details in Gourgue et al (2009)
+
+from thetis import *
+import pytest
+import math

```

Comment:
`math` is not used. In general `numpy` is imported as `np` so you don't need this.